risk-to-severity-mapping/README.md

```diff
@@ -0,0 +1,59 @@
+# Script to Map Severities to Risk Scores
+
+To learn more on [GitGuardian risk score](https://docs.gitguardian.com/releases/saas/2025/12/17/2-changelog)
+
+
+## Severity Mapping (editable)
+
+```bash
+SEVERITY_MAPPING = {
+    "critical": 80,  # Risk score >= 80
+    "high": 60,      # Risk score >= 60 and < 80
+    "medium": 40,    # Risk score >= 40 and < 60
+    "low": 20,       # Risk score >= 20 and < 40
+    "info": 0,       # Risk score < 20
+}
+```
+[GitGuardian_API_KEY creation](https://docs.gitguardian.com/api-docs/personal-access-tokens) use SATs for higher rate-limits.
```

Comment:
Add the min scope needed for the token creation which should be Incidents:write, Incidents:read.